tests/comb_to_cggi/add_one.mlir

```diff
@@ -1,38 +1,57 @@
-            %16 = comb.truth_table %11, %ARG05, %ARG06 -> 128 : ui8
-            %18 = comb.truth_table %16, %ARG07, %FALSE -> 6 : ui8
-            secret.yield %18, %15, %13, %10, %8, %5, %2, %3 : i1, i1, i1, i1, i1, i1, i1, i1
-        } -> (!secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>)
-  // CHECK: return
-    func.return %0#0, %0#1, %0#2, %0#3, %0#4, %0#5, %0#6, %0#7 : !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>, !secret.secret<i1>
+    %c7 = arith.constant 7 : index
+    %c6 = arith.constant 6 : index
+    %c5 = arith.constant 5 : index
+    %c4 = arith.constant 4 : index
+    %c3 = arith.constant 3 : index
+    %c2 = arith.constant 2 : index
+    %c1 = arith.constant 1 : index
+    %c0 = arith.constant 0 : index
+    %0 = secret.cast %arg0 : !secret.secret<i8> to !secret.secret<tensor<8xi1>>
+    %1 = secret.generic ins(%0 : !secret.secret<tensor<8xi1>>) {
+    ^bb0(%arg1: tensor<8xi1>):
+      %extracted = tensor.extract %arg1[%c0] : tensor<8xi1>
+      %3 = comb.truth_table %extracted, %true, %false -> 8 : ui8
+      %extracted_0 = tensor.extract %arg1[%c1] : tensor<8xi1>
+      %4 = comb.truth_table %3, %extracted_0, %false -> 150 : ui8
+      %5 = comb.truth_table %3, %extracted_0, %false -> 23 : ui8
+      %extracted_1 = tensor.extract %arg1[%c2] : tensor<8xi1>
+      %6 = comb.truth_table %5, %extracted_1, %false -> 43 : ui8
+      %extracted_2 = tensor.extract %arg1[%c3] : tensor<8xi1>
+      %7 = comb.truth_table %6, %extracted_2, %false -> 43 : ui8
+      %extracted_3 = tensor.extract %arg1[%c4] : tensor<8xi1>
+      %8 = comb.truth_table %7, %extracted_3, %false -> 43 : ui8
+      %extracted_4 = tensor.extract %arg1[%c5] : tensor<8xi1>
+      %9 = comb.truth_table %8, %extracted_4, %false -> 43 : ui8
+      %extracted_5 = tensor.extract %arg1[%c6] : tensor<8xi1>
+      %10 = comb.truth_table %9, %extracted_5, %false -> 105 : ui8
+      %11 = comb.truth_table %9, %extracted_5, %false -> 43 : ui8
+      %extracted_6 = tensor.extract %arg1[%c7] : tensor<8xi1>
+      %12 = comb.truth_table %11, %extracted_6, %false -> 105 : ui8
+      %13 = comb.truth_table %extracted, %true, %false -> 6 : ui8
+      %14 = comb.truth_table %5, %extracted_1, %false -> 105 : ui8
+      %15 = comb.truth_table %6, %extracted_2, %false -> 105 : ui8
+      %16 = comb.truth_table %7, %extracted_3, %false -> 105 : ui8
+      %17 = comb.truth_table %8, %extracted_4, %false -> 105 : ui8
+      %from_elements = tensor.from_elements %12, %10, %17, %16, %15, %14, %4, %13 : tensor<8xi1>
+      secret.yield %from_elements : tensor<8xi1>
+    } -> !secret.secret<tensor<8xi1>>
+    // CHECK: [[CONCAT:%.*]] = tensor.from_elements
+    %26 = secret.cast %25 : !secret.secret<tensor<8xi1>> to !secret.secret<i8>
```

Comment:
```suggestion
    %2 = secret.cast %1 : !secret.secret<tensor<8xi1>> to !secret.secret<i8>
```

Reply:
Ohhh nice thank you for catching this really crazy copy/paste error I made (I copy pasted the CHECKs from the previous IR but didn't update this line between the CHECKs)